app/models/pillar.rb

```diff
@@ -41,7 +41,8 @@ def all_pillars
         ldap_admin_group_name:   "ldap:admin_group_name",
         ldap_tls_method:         "ldap:tls_method",
         ldap_mail_attribute:     "ldap:mail_attribute",
-        cloud_framework:         "cloud:framework"
+        cloud_framework:         "cloud:framework",
+        registries:              "docker:registries"
```

Comment:
If we are using our own models I don't see why we want this here, since the `Pillar` model won't save this information but `Certificate` and `DockerRegistry` will. I think we can remove this and serve this content using the `InternalApi::V1::PillarsController` to include `DockerRegistry`'s.